### ISSUE TITLE
Access form elements by field name

### ISSUE THREAD

**Issue by RupertAvery** [stephanrauh/ngx-extended-pdf-viewer#1545]
I would like to be able to add a highlight to form elements by adding CSS classes, or bring the element into focus, by using the field name assigned to the form.

Is there existing functionality to do this? to be able to access the element by its field name?

**Comment by stephanrauh** [comment]
Well, form support is where my library get ugly. Generally speaking, there's no connection between the form fields and the form names - unless the author of the PDF file takes care of using sensible names. Otherwise, you might end up with random names.

Let's assume the PDF author choose recognizable form names.

Then the names become the field names in the HTML DOM. More to the point: you can use a query like 
```js
const field = document.querySelector("input[name='" + key + "']") as HTMLElement;
```
to find the DOM element. Angular developers don't like this kind of direct access for a good reason, but in this case, it's OK. pdf.js doesn't know anything about Angular, so it's using native DOM manipulation, anyways.

Once you've got the DOM element, you can do anything. You just need to take in mind that the library may dispose hidden pages, and that it tends to render pages lazily. Whatever you do, register the event `(pageRendered)` or `(textLayerRendered)`. It'd even better to register `(annotationLayerRendered)`, but I haven't implemented this (see #1536).

The screenshot give you an idea how the DOM tree is organized. Pages contain of several layers, the annotation layer being one of them. Forms are rendered in the annotation layer. I've also highlighted one of the input fields:
<img width="640" alt="image" src="https://user-images.githubusercontent.com/3045767/192875228-a23f58fe-6bef-4b34-b65a-3f6bc62f0c04.png">

**Comment by stephanrauh** [comment]
I'm positive I've answered your question, so I'm closing this ticket now. Don't hesitate to add comments below if you've still got questions.

Enjoy!
Stephan
